hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/lock/OzoneLockStrategy.java

```diff
@@ -31,19 +31,16 @@
  */
 public interface OzoneLockStrategy {
   boolean acquireWriteLock(OMMetadataManager omMetadataManager,
-                           OzoneManagerLock.Resource resource,
                            String volumeName, String bucketName, String keyName)
```

Comment:
@tanvipenumudy IMO, Exception throws clause is not required in the release function/API. In general, releasing logic should handle all the exceptions within itself otherwise it can leads to leaking of resources. For example, what happens if any exception thrown and lock is not released, which can leads to resource leaks, isn't it.

Could you please remove the `throws IOException` from the release lock interface and its impl classes. Thanks!

```
OzoneLockStrategy.java 

  void releaseWriteLock(OMMetadataManager omMetadataManager,
                        OzoneManagerLock.Resource resource, String volumeName,
                        String bucketName, String keyName);

  void releaseReadLock(OMMetadataManager omMetadataManager,
                       OzoneManagerLock.Resource resource, String volumeName,
                       String bucketName, String keyName);
```

Reply:
Thank you for the suggestion, made the necessary changes!